src/webrtc/call.ts

```diff
@@ -84,32 +84,73 @@ enum CallParty {
     REMOTE = 'remote',
 }
 
+enum MediaQueueId {
+    REMOTE_VIDEO = 'remote_video',
+    REMOTE_AUDIO = 'remote_audio',
+    LOCAL_VIDEO = 'local_video',
+}
+
 enum CallErrorCode {
     /** An error code when the local client failed to create an offer. */
-    ERR_LOCAL_OFFER_FAILED = "local_offer_failed",
+    LOCAL_OFFER_FAILED = 'local_offer_failed',
```

Comment:
same re PascalCasing

Reply:
These ones are part of API unfortunately so need to stay for backwards-compat

Reply:
oh wait, you mean the enum names not the values